db/re/skill_db.yml

```diff
@@ -19423,7 +19421,7 @@ Body:
     CastCancel: true
     CastTime: 3000
     AfterCastActDelay: 500
-    Duration1: 15000
+    Duration1: 13000
```

Comment:
Maybe suspicious.

Reply:
Cleaned up as Hell Inferno no longer inflicts Burning after the Warlock Improvement patch.